ui/app/AppLayouts/Wallet/panels/DAppsWorkflow.qml

```diff
@@ -153,8 +155,14 @@ ConnectedDappsButton {
             Connections {
                 target: root.wcService.requestHandler
 
-                function onMaxFeesUpdated(maxFees, symbol) {
+                function onMaxFeesUpdated(maxFees, maxFeesWei, haveEnoughFunds, symbol) {
                     maxFeesText = `${maxFees.toFixed(2)} ${symbol}`
+                    var ethStr = "?"
+                    if (globalUtils) {
+                        ethStr = globalUtils.wei2Eth(maxFeesWei, 9)
+                    }
+                    maxFeesEthText = `${ethStr} ETH`
```

Comment:
Anything against using the standard `LocaleUtils.formatFoo` here?

Reply:
Not sure what you mean. I found only the following function in `LocaleUtils`: `formatDate`, `formatTime`, `formatDateTime` and `formatRelativeTimestamp`.
This code is temporary to have a placeholder until the implementation is finalized for the data source in here [#15190](https://github.com/status-im/status-desktop/issues/15190), [#15191](https://github.com/status-im/status-desktop/issues/15191)